packages/wdio-sync/tests/executeHooksWithArgs.test.js

```diff
@@ -0,0 +1,36 @@
+import Fiber from '../src/fibers'
```

Comment:
I made some changes to `executeHooksWithArgs.js` and added a test for it as well in my PR https://github.com/webdriverio/webdriverio/pull/4354/files#diff-a4452c2cccf4940e8a09d260fa8333db

Reply:
Should I rebase with your PR or wait for it to be merged?

Reply:
I'm fine with merging it as is.